docs/_extensions/napoleon_django/docstring.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.ext.napoleon.docstring
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+    Classes for docstring parsing and formatting.
+
+
+    :copyright: Copyright 2007-2016 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from sphinx.ext.napoleon.docstring import GoogleDocstring
+
+
+class DjangoGoogleDocstring(GoogleDocstring):
+    """Add support for Django-specific sections to napoleon's GoogleDocstring parser.
+
+    Parameters
+    ----------
```

Comment:
Napoleon does numpy and google. Since this is rudely copied from napoleon itself, I thought it made sense to keep the same docstring style as the source.